### PR TITLE
clarify required parameters [TENJIN-9382]

### PR DESCRIPTION
**Pull request by makototaguchi1110**
**JIRA**:

- https://adromance.atlassian.net/browse/TENJIN-9382

**Proposed Changes:**

- Description

**Steps:**

- [ ] Reviewed
- [ ] Deployed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -366,14 +366,11 @@ boolean CheckOptInValue()
...
-- Please note that we require the following parameters to properly track devices in Tenjin's system:
+- Please note that we require the following parameters to properly track devices in Tenjin's system. If one of these mandatory parameters is missing the event will not be processed or recorded.
 
   - `ip_address`
-  - `advertising_id`
-  - `limit_ad_tracking`
-  - `referrer` (Android)
-  - `iad` (iOS)
-
+  - `advertising_id` (Android), `advertising_id` and/or `developer_device_id` (iOS)
```
Comment (vanner) [inline README.md:372]: `advertising_id` is listed twice...is that a typo?

Comment (makototaguchi1110) [reply README.md]: Ok, I updated to separate the list for android and ios.. hope this is better.

File: README.md [hunk 2]
```diff
@@ -366,15 +366,16 @@ boolean CheckOptInValue()
...
-  - `advertising_id`
-  - `limit_ad_tracking`
-  - `referrer` (Android)
-  - `iad` (iOS)
-
-- If you are targeting IMEI and/or OAID Ad Networks, add:
+- Please note that we require the following parameters to properly track devices in Tenjin's system. If one of these mandatory parameters is missing the event will not be processed or recorded.
+
+  - For Android, 
+    - `ip_address`
```
Comment (amirmanji) [inline README.md:372]: Does IP address needs to be listed as a parameter? It's inherent in the requests we receive.

Comment (makototaguchi1110) [reply README.md]: thanks, updated

**Review by hc5duke — APPROVED** [review]

**Pull request merged**